src/docfx/validation/ContentValidator.cs

```diff
@@ -59,6 +64,7 @@ public void ValidateImageLink(FilePath file, SourceInfo<string> link, MarkdownOb
                         IsInlineImage = origin.IsInlineImage(imageIndex),
                         SourceInfo = link.Source,
                         ParentSourceInfoList = origin.GetInclusionStack(),
+                        Monikers = origin.GetZoneLevelMonikers().ToList(),
```

Comment:
Are you able to change the contract to `IReadOnlyList<string>` where possible to avoid these `ToList` calls?

Reply:
I have updated PR to Docs.Validation which is currently pending review. Will change this accordingly later.